root/io/stdpair/CMakeLists.txt

```diff
@@ -10,11 +10,11 @@ ROOT_GENERATE_DICTIONARY(G__CmsPair ${CMAKE_CURRENT_SOURCE_DIR}/cmspair.h LINKDE
 ROOTTEST_LINKER_LIBRARY(CmsPair TEST G__CmsPair.cxx LIBRARIES ${ROOT_LIBRARIES})
 
 ROOT_ADD_TEST(roottest-root-io-stdpair-collection-build 
-              COMMAND ${CMAKE_COMMAND} --build ${CMAKE_BINARY_DIR} --config $<CONFIG> --target CmsPairCollection${fast}
+              COMMAND ${CMAKE_COMMAND} --build ${CMAKE_BINARY_DIR} --config $<CONFIG> --target G__CmsPairCollection${fast}  CmsPairCollection${fast} -- ${always-make}
```

Comment:
Where is `${always-make}` being set? Is `-` an allowed character for a CMake variable name?